README.md

```diff
@@ -44,7 +44,7 @@ cd kotlin
 ### Swift
 
 ```bash
-cargo swift package -p ios -n MobileSdkRs
+cargo swift package -p ios -n MobileSdkRs --release
```

Comment:
May I ask if using a release build is still needed with the latest version of uniffi/cargo-swift/etc?

Reply:
I had some runtime errors while using the w3c-vc-barcodes through uniffi, and they disappeared when I compiled with the release flag

Reply:
`cargo-swift` recently had a patch verison released, and we're not using the latest `uniffi` version. Do you think you would have time to test again with both of these at their latest version? Otherwise, put a comment in the README to explain why we need `--release`

Reply:
Tested with `cargo-swift@0.8.1` and `uniffi@0.28.1`, the error is still there. I'll commit bumping the uniffi version and adding a note to the readme explaining why we are using the `--release` flag